x-pack/test/apm_api_integration/common/config.ts

```diff
@@ -126,22 +126,10 @@ export function createTestConfig(config: ApmFtrConfig) {
         },
         ml: MachineLearningAPIProvider,
         // legacy clients
-        legacySupertestAsNoAccessUser: getLegacySupertestClient(
-          kibanaServer,
-          ApmUsername.noAccessUser
-        ),
-        legacySupertestAsApmReadUser: getLegacySupertestClient(
-          kibanaServer,
-          ApmUsername.viewerUser
-        ),
         legacySupertestAsApmWriteUser: getLegacySupertestClient(
```

Comment:
Could you also remove `legacySupertestAsApmWriteUser` and `legacySupertestAsApmReadUserWithoutMlAccess` which don't seem to be used? This way we can also get rid of `getLegacySupertestClient`

Reply:
I'm planning to address those two in another PR